editor/changeset_wrapper.cpp

```diff
@@ -20,25 +22,31 @@ namespace osm
 {
 
 ChangesetWrapper::ChangesetWrapper(TKeySecret const & keySecret,
-                                   ServerApi06::TKeyValueTags const & comments)
-  : m_changesetComments(comments),
-    m_api(OsmOAuth::ServerAuth().SetToken(keySecret))
+                                   ServerApi06::TKeyValueTags const & comments) noexcept
+  : m_changesetComments(comments), m_api(OsmOAuth::ServerAuth(keySecret))
 {
 }
 
-ChangesetWrapper::~ChangesetWrapper()
+ChangesetWrapper::~ChangesetWrapper() noexcept
```

Comment:
Note: 
> Any user-defined destructor is noexcept(true) by default, unless the declaration specifies 
> otherwise, or the destructor of any base or member is noexcept(false).